### DISCUSSION TITLE
Opening browser outside of sandbox removes all sandboxed tabs

### DISCUSSION THREAD

**Discussion by Gr4cchus** [sandboxie-plus/Sandboxie#1225] [Q&A Feedback] [answered]
~~When running sandboxie and closing the window with tabs then opening the browser again normally without sandboxie seems to remove all the tabs or place you were at in sandboxie? Any idea how to restore the tabs back, or what it deletes those tabs?~~
Google chrome tabs in sandboxie session are lost when running google chrome normally without sandboxie. How do you prevent this, or restore that session/tabs inside sandboxie? `ctrl+shift+t` doesnt restore anything like it normally does when this happens.

**Comment by isaak654** [comment]
Sandboxie provides compatibility templates that allow direct access to the profile folder of different browsers.
![compatibility_templates](https://user-images.githubusercontent.com/12372772/134764931-ae758a8b-8e02-4635-b9cc-2c58963a445b.png)

It's recommended to check the profile path of your applied template in the `Templates.ini` file (on your Sandboxie installation folder) -> `[TemplateSettings]` section.
If you have a different profile folder than the default profile path set on that file, you can customize it from the `Template Folders` tab that is displayed next to the `Compatibility Templates` tab. The condition to see a profile folder to change there is to apply an existing browser template, press ok to confirm the choice and return to the `Sandbox Options`. Only after doing that, you will see the applied browser listed in `Template Folders`  with a template folder to change.

Alternatively, you can also define a custom rule like this in `C:\Windows\Sandboxie.ini`, under your box like `[DefaultBox]`:
`OpenFilePath=msedge.exe,C:\Users\Test\AppData\Local\Microsoft\Edge\User Data\Profile2\*`
This is the equivalent of setting a Resource Access rule in the user interface.

If you need an easier access to the ini file on Sandboxie Plus:

> 
>     Go to Options menu -> Edit ini file -> confirm yes
>     Add that line below [DefaultBox] (if you intend to use this sandbox to run the program)
>     Save the ini file and close it

**Reply by Gr4cchus** [reply]
I might have to edit my post. To clarify, the tabs are being removed from the sandboxie instance anytime the brower isnt run with sandboxie. This sounds like it does the opposite or something.

**Reply by isaak654** [reply]
See my answer below: https://github.com/sandboxie-plus/Sandboxie/discussions/1225#discussioncomment-1383671

**Comment by isaak654** [comment, marked as answer]
So if I understand correctly, you may be interested in the following open issue: [Chrome does not restore tabs if ran out of Sandbox](https://github.com/sandboxie-plus/Sandboxie/issues/558)

AFAIK there is no fix yet...

**Reply by Gr4cchus** [reply]
yeah, without looking this sounds exactly like the issue. Is there maybe some extension that can manually do this, to save sessions?

**Reply by isaak654** [reply]
I don't know, but you can continue there in order to avoid duplication.
